.gitignore

```diff
@@ -1,7 +1,6 @@
 .DS_Store
 .gitignore
 node_modules
-public
```

Comment:
Removing this will make non-docker users sync build artifacts to their git repository.

Reply:
(which also fulfills the role of `.gitkeep`)

Reply:
Updated, removed use of .gitignore under `content`.